include/deal.II/lac/block_linear_operator.h

```diff
@@ -457,6 +457,90 @@ lower_triangular_operator(const BlockMatrix &block_matrix)
 /**
  * @relates LinearOperator
  *
+ * This function is a specification of the above function that
+ * allows to work with std::array of std::array of LinearOperator
+ *
+ * @ingroup LAOperators
+ */
+
+template <unsigned int n,
+          typename Range = BlockVector<double>,
+          typename Domain = Range>
+LinearOperator<Range, Domain>
+lower_triangular_operator(const std::array<std::array<LinearOperator<typename Range::BlockType, typename Domain::BlockType>, n>, n> &);
+
+
+template <unsigned int n, typename Range, typename Domain>
+LinearOperator<Range, Domain>
+lower_triangular_operator(const std::array<std::array<LinearOperator<typename Range::BlockType, typename Domain::BlockType>, n>, n> &block_matrix)
```

Comment:
Since this implementation is a verbatim copy of the above one, can you remove the one above, and replace it with a call to this? Make an array out of a block matrix, and call this function with the array just created. This avoids code replication...